scalableminds/webknossos
# in task deletion, also delete its annotations

### URL of deployed dev instance (used for testing):
- https://deletetaskannotations.webknossos.xyz

### Steps to test:
- create task
- request it, leave it active
- delete the task in the admin view
- task list in dashboard should not throw error, but rather show your other tasks, but no longer that one

### Issues:
- fixes #2873

------
